<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>661</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QLabel" name="nameCinema">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Название кино</string>
    </property>
   </widget>
   <widget class="QLabel" name="pici">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>266</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Картинка</string>
    </property>
   </widget>
   <widget class="QLabel" name="genre">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Жанр</string>
    </property>
   </widget>
   <widget class="QLabel" name="year">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Год</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>150</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Смотреть</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
